microcosm_postgres/migrate.py

```diff
@@ -0,0 +1,203 @@
+"""
+Alembic CLI with simplified configuration.
+
+Alembic is a terrific tool that makes some unfortunate choices about configuration,
+expecting a verbose directory structure with several layers of configuration.
+
+This module monkey patches Alembic's CLI tool to work better within a microcosm.
+
+To use this entry enty point instead of the Alembic CLI:
+
+ 0. Don't use `alembic init`
+
+ 1. Write models that extend `microcosm_postgres.models.Model`
+
+ 2. Add a `migrations` directory within your source tree.
+
+    This directory does not need to be an importable Python module, but it should
+    be included as part of your distribution so that migrations ship with the service.
+
+ 3. Initialize your object graph (including your models) and ensure that it knows the
+    migrations directory path.
+
+    For example:
+
+        from microcosm_postgres.example import Company, Employee  # noqa
+        graph = create_object_graph(name="example", testing=True)
+        graph.migrations_dir = join(dirname(__file__), "migrations")
```

Comment:
does pkg_resources (specifically .resource_filename() or so) help in anyway here to be able to access content programmatically without special knowledge or assumptions? e.g https://pythonhosted.org/setuptools/pkg_resources.html#id22